port/unix/omrsysinfo.c

```diff
@@ -2681,6 +2682,13 @@ searchSystemPath(struct OMRPortLibrary *portLibrary, char *filename, char **resu
 
 #endif /* defined(AIXPPC) || defined(J9ZOS390) */
 
+/* Cgroup v1 cpu files */
+#define CGROUP_CPU_CFS_QUOTA_US_FILE "cpu.cfs_quota_us"
+#define CGROUP_CPU_CFS_PERIOD_US_FILE "cpu.cfs_period_us"
+
+/* Cgroup v2 cpu files */
+#define CGROUP_CPU_MAX_FILE "cpu.max"
```

Comment:
This should be moved to the start of the file where other cgroup macros are defined. At the start, cgroup macros are also wrapped with `defined(LINUX)`.

Reply:
Should https://github.com/eclipse/omr/pull/6465/files#diff-6a11627a20d4ac8347fc9f838e7b4d6aa0df34cac472236196291815b5fc7db5R3044-R3063 be moved as well then?

Reply:
yes. missed those ones.